docs/.vitepress/config.mts

```diff
@@ -5,41 +5,44 @@ const PATH_FEATURES = '/features/';
 
 // https://vitepress.dev/reference/site-config
 export default withMermaid({
-  title: "Wahllokalsystem",
-  description: "Datenerfassung am Wahltag",
-  lang: "de-DE",
-  base: '/Wahllokalsystem/',
-  srcDir: 'src', //markdown files are located in that directory
-  themeConfig: {
-    // https://vitepress.dev/reference/default-theme-config
-    nav: [
-      { text: 'Home', link: '/' },
-      { text: 'About', link: '/about/' },
-      { text: 'Features', link: '/features/' },
-      { text: 'Technik', link: PATH_TECHNIK }
-    ],
+    title: "Wahllokalsystem",
```

Comment:
hier müssen wir schauen wie wir mit der Formatierung umgehen. Ich hatte die Autoformatierung in IntelliJ deaktiviert dafür.

Ich schlage vor ein Issue erstellen damit wir hier prettier einbinden.

Reply:
prettier nur für die Doku oder für alles?

Reply:
erst einmal für die Doku. Für Backend müssen sollten wir uns gesondert ansehen. Aktuell haben dort zumindest mit checkstyle und spotless einiges.

Reply:
Siehe #212